meteor/sofie-server/data-access/repositories/interfaces/part-repository.ts

```diff
@@ -2,4 +2,5 @@ import { Part } from '../../../model/entities/part'
 
 export interface PartRepository {
 	getParts(segmentId: string): Promise<Part[]>
+	deleteSegmentParts(segmentId: string): Promise<void>
```

Comment:
`SegmentParts` indicate that we have a concept called `SegmentPart` which we don't.
As talked about in a different comment, if you want to state everything in the name, then I would highly recommend `deletePartsForSegment`.

Reply:
Have applied your suggested naming scheme to the method names. (`delete<children>For<parent>`)